ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -395,6 +397,12 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             ConfigDef.Importance.LOW,
             "Enable the security manager for UDFs. Default is true and will stop UDFs from"
                + " calling System.exit or executing processes"
+        ).define(
```

Comment:
Why do we want a config for this? Sort of a coarse access control mechanism?

Reply:
Yes - if you are deploying this in production in interactive mode you may want to disable the ability for users to arbitrarily insert into topics (the security story around this isn't great as it stands, though with the changes being made to support custom security plugins this might get better)